src/averageAges.js

```diff
@@ -15,11 +15,28 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm');
+
+  const menFromNeededCentury = men.filter(person =>
+    Math.ceil(person.died / 100) === century
+  );
```

Comment:
do it only if century was passed as a param, use ternary operator for that, like `century ? yourfilter : men`